counting_sort/counting_sort.py

```diff
@@ -0,0 +1,30 @@
+def countOccurences(a, c):
+    for i in range(len(a)):
+        c[a[i]] += 1
+    for i in range(1, len(c)):
+        c[i] = c[i - 1] + c[i]
+
+
+def countingSort(a, k):
```

Comment:
Change name of functions as per underscore rule, `camelCase` is only followed in Java.